DDOC-870: Remove obsolete indexing requirements (#495)
box/developer.box.com: content/guides/metadata/5-queries/7-comparison.md

| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | What is indexed?  | This API only return files/folders based on the values in the metadata templates that are searched.                                                                                                                                                                                                                  | This API returns files, folders and web links based on values in the item names, descriptions, contents (up to the first 10,000 bytes) as well as the associated metadata template instances.                                                                                                                                                            |

Reviewer: ```suggestion
This API only returns files/folders based on the values in the metadata templates that are searched.                                                                                                                                                                                                                  ```

> | What is indexed?  | This API only return files/folders based on the values in the metadata templates that are searched.                                                                                                                                                                                                                  | This API returns files, folders and web links based on values in the item names, descriptions, contents (up to the first 10,000 bytes) as well as the associated metadata template instances.                                                                                                                                                            |

Reviewer: ```suggestion
| What is indexed?  | This API only returns files/folders based on the values in the metadata templates that are searched.                                                                                                                                                                                                                  | This API returns files, folders and web links based on values in the item names, descriptions, contents (up to the first 10,000 bytes) as well as the associated metadata template instances.                                                                                                                                                            |
```

> | Indexing time     | This API will return accurate results as soon as metadata has been added, removed, updated or deleted for a file or folder.                                                                                                                                                                                          | This API us subject to a search indexing delay, which is typically 10 minutes yet may be longer in some cases. This means that items may not be returns for more than 10 minutes after metadata has been updated.                                                                                                                                        |

Reviewer: ```suggestion
| Indexing time     | This API returns accurate results as soon as metadata has been added, removed, updated or deleted for a file or folder.                                                                                                                                                                                          | This API is subject to a search indexing delay, which is typically 10 minutes, yet may be longer in some cases. This means that items may not be returned for more than 10 minutes after metadata has been updated.                                                                                                                                        |
```
